add get_started_with_Markdown (#10)
ttttLarva/Larva: docs/github/05_get_started_with_Markdown.md

> #  快速上手Markdown

Reviewer: 中英文之间、数字和中文之间保持空格。其它地方自己检查下。

Reviewer 2: 收到，已检查

<img src="image/ball.png" style="zoom:13%;" align="left" />**Bilibili视频传送门：**[快速上手Markdown](https://www.bilibili.com/video/BV1P44y1a7oB?spm_id_from=333.999.0.0)  <img src="image/ball.png" style="zoom:13%;"/>

> ## 什么是 Markdown
> Markdown是一种标记性语言，在GitHub上提交issue，PR，默认都使用的是Markdown。

Reviewer: ```suggestion
## 什么是 Markdown

Markdown是一种标记性语言，在GitHub上提交issue，PR，默认都使用的是Markdown。
```

标题后要有空行，其它地方自己检查下。

Reviewer 2: 已在标题后添加空行

> Markdown简单易学，展现内容的手段丰富多样，可以参与开源，也可以作为自己的笔记。

Reviewer: ```suggestion
Markdown 简单易学，展现内容的手段丰富多样，可以参与开源，也可以用来记录自己的笔记。
```

Reviewer 2: 已修改

## 在GitHub上写Markdown

> 接下来我将以深度学习框架`OneFlow`的issue为例子， 展现一下Markdown的功能。

Reviewer: OneFlow 那里，带上 OneFlow 的仓库地址吧

Reviewer 2: 已带上OneFlow仓库地址

#  快速上手Markdown

<img src="image/ball.png" style="zoom:13%;" align="left" />**Bilibili视频传送门：**[快速上手Markdown](https://www.bilibili.com/video/BV1P44y1a7oB?spm_id_from=333.999.0.0)  <img src="image/ball.png" style="zoom:13%;"/>

## 什么是 Markdown 

> Markdown 是一种标记性语言，在 GitHub 上提交 issue，PR，默认都使用的是 Markdown。

Reviewer: ```suggestion
Markdown 是一种标记性语言，在 GitHub 上提交 issue、PR，默认都使用的是 Markdown。
```

Reviewer 2: 已修改

Markdown 简单易学，展现内容的手段丰富多样，可以参与开源，也可以用来记录自己的笔记。

> ## 在GitHub上写Markdown

Reviewer: 空格问题

Reviewer 2: 已修改

> 
> 

Reviewer: ```suggestion
```

Reviewer 2: 我发现在源代码中空行数量和实际预览中不同，如果去掉预览时标题下就没有空行了。

> 接下来我将以深度学习框架 [oneflow](https://github.com/Oneflow-Inc/oneflow) 的 issue 为例子， 展现一下 Markdown 的功能。

Reviewer: ```suggestion
接下来我将以深度学习框架 [OneFlow](https://github.com/Oneflow-Inc/oneflow) 的 issue 为例子， 展现一下 Markdown 的功能。
```

OneFlow 作为产品，是首字母大写的

Reviewer 2: 已进行修改

>  我们可以看到在下面这个 issue 中，既有图片也有代码区，还有列表，这些是如何用 Markdown 实现的呢？

Reviewer: 加上 issue 的 URL，这样体验更好。
如果找不到下图中对应的 URL 了，就换一个有代表性的，同时更新下图。

Reviewer 2: 已添加 issue 的 URL。

![markdown_issue.png](image/markdown_issue.png)

> 我们在`OneFlow`中创建一个新的issue，点击 New issue 按钮。

Reviewer: 给 URL 链接。以及空格问题

要主动思考下，怎样的产出，是方便观众的。

Reviewer 2: 已添加 URL 链接。

<img src="image/new_issue.png" alt="new_issue.png" style="zoom:80%;" />

> 在 Write 和 Preview 部分就是Markdown书写区和预览区的一个对比。

Reviewer: 空格问题

Reviewer 2: 已修改

<img src="image/write.png" style="zoom:25%;"  align="left"  /><img src="image/preview.png" style="zoom:25%;" lign="right"/>

- `## ` 加空格后面所对应的内容就是标题

- 连接符 `- ` 加空格后面的内容是列表所对应的一个 item

> 

Reviewer: 这里太生硬直接过去了。最好加句过渡。比如，我们接下来看看常见的 Markdown 语法有哪些，系统学习下。

Reviewer 2: 已添加过渡句。

> ## 学习Markdown语法

Reviewer: 空格问题

Reviewer 2: 已修改

### 1. 标题

在想要设置为标题的文字前面加 `# ` 来表示
一个 `# ` 是一级标题，两个 `# ` 是二级标题，以此类推。支持六级标题。

```
# 这是一级标题
## 这是二级标题
### 这是三级标题
#### 这是四级标题
##### 这是五级标题
###### 这是六级标题
```

### 2. 图片

语法：

```
![图片alt](图片地址 ''图片title'')
```

图片 alt 就是显示在图片下面的文字，相当于对图片内容的解释。
图片 title 是图片的标题，当鼠标移到图片上时显示的内容。title 可加可不加。

示例：

```
![image](https://user-images.githubusercontent.com/54847470/154951006-a7dd7692-597e-4f9f-a33c-712219f0c5a7.png)
```

效果：

<img src="https://user-images.githubusercontent.com/54847470/154951382-58059c14-db4d-41a5-9734-4f6334305a57.png" alt="image" style="zoom:25%;"  align="center"  />

### 3. 超链接

语法：

```
[超链接名](超链接地址 "超链接title")
```

title 可加可不加

示例：

```

> [百度](http://baidu.com)

Reviewer: 把百度的例子换成 ttlarva.com，或者 oneflow 吧

Reviewer 2: 已换成 OneFlow 的地址。

```

效果：

[百度](https://baidu.com)

### 4. 代码

单行代码：代码之间分别用一个反引号包起来

```
`代码内容`
```

代码块：代码之间分别用三个反引号包起来，且两边的反引号单独占一行，在第一行引号后面加入代码种类可以获得代码高亮。

````python
```Python
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashtable = dict()
        for i in range(len(nums)):
            if target - nums[i] in hashtable:
                return [i, hashtable[target - nums[i]]]
            hashtable[nums[i]] = i
```
````

### 5. 引用

`>`表示引用，表示当前语句不是自己的，在开源项目中常常用于回复别人的话。

示例：

```

> >C++很好用

Reviewer: ```suggestion
> C++很好用
```

是不是少了空格

Reviewer 2: 已添加空格

```

效果：

> C++很好用

除了上述语法外，Markdown还有很多用法，大家可以在扩展阅读中学习。

> ## VS插件分享

Reviewer: ```suggestion
## VS Code 插件分享
```

VS 一般指 Visual Stdudio，和 Visual Studio Code 还不是一个东西

Reviewer 2: 已将 VS 修改成 VS Code

> 这里分享一个非常实用的插件，在使用 Markdown 时，**Auto-Open Markdown Preview** 插件可以帮我们实时渲染 Markdown 页面。

Reviewer: 可以给个 URL

Reviewer 2: 已添加 Auto-Open Markdown Preview 插件的 URL

<img src="image/Auto-Open Markdown Preview.png" alt="Auto-Open Markdown Preview.png" style="zoom: 50%;" />

## 扩展阅读

> 

Reviewer: 删除不必要的多余行

> - [GitHub官方Markdown教程](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

Reviewer: ```suggestion
- [GitHub 官方 Markdown 教程](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
```

Reviewer 2: 已添加空格
